holoviews/plotting/plot.py

```diff
@@ -582,6 +595,9 @@ class GenericElementPlot(DimensionedPlot):
     bgcolor = param.ClassSelector(class_=(str, tuple), default=None, doc="""
         If set bgcolor overrides the background color of the axis.""")
 
+    default_span = param.ClassSelector(default=2.0, class_=(int, float, tuple), doc="""
+        Default span if the spans zero range.""")
```

Comment:
I'm not quite sure that a `span` is supposed to be...

Reply:
Doesn't help that the current docstring isn't english...

Basically it defines what happens if the data has no range, e.g. when you have a single point like ``hv.Points([(0, 0)])`` it means the axes will span from -1 to 1 (i.e. a span of 2).

Reply:
I see. I would like to have a better name for this concept but I don't have one to suggest at the moment. Conceptually it is something like `fallback_span` but that name is horrible...

Reply:
a fallback_range maybe?  Is this triggered only if the axis extent is [a,a], i.e., if the span is 0?
I could think of a tolerance, e.g. when plotting residuals (with bounds [-1e-12, 1e-8] for example).

Looking at this, I used the terms range, extent, bounds....

Reply:
I used ``default_span`` because that's what bokeh uses, ``fallback_span`` is semantically okay but I also don't like the name. ``fallback_range`` is incorrect because it's not a range or extent since it specifies the span or interval not explicit upper and lower bounds.